lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -81,9 +81,11 @@ def problem_scores(self):
         ):
             block = self.course_data.structure[block_key]
             if getattr(block, 'has_score', False):
-                locations[block_key] = get_score(
+                problem_score = get_score(
                     submissions_scores={}, csm_scores={}, persisted_block=None, block=block,
                 )
+                if problem_score:
```

Comment:
Good question! Objects can evaluate to False: https://docs.python.org/2/reference/datamodel.html#object.__nonzero__